ios/Cartfile

```diff
@@ -2,3 +2,4 @@ github "marmelroy/Zip"
 github "DaveWoodCom/XCGLogger" ~> 6.1.0
 github "dennisweissmann/DeviceKit" ~> 2.3
 github "ashleymills/Reachability.swift"
+github "getsentry/sentry-cocoa" ~> 4.4.1
```

Comment:
Why not use `4.4.3` like in Cartfile.resolved?

Reply:
Because carthage has more recent info than [their documentation](https://docs.sentry.io/clients/cocoa/#carthage).  And it doesn't hurt anything; when present, Cartfile.resolved overrides Cartfile.